test/configCases/parsing/issue-6867/index.js

```diff
@@ -0,0 +1,57 @@
+it("should compile default export unnamed function declaration", function() {
```

Comment:
Could you move this test case from `configCases` to `cases`?

Reply:
done